apps/webapp/mix.lock

```diff
@@ -0,0 +1,35 @@
+%{"absinthe": {:hex, :absinthe, "1.1.10", "ad8b34f2ee85ec2d21ba7dac0c50fa207f466799301e475f4bc1e48f73c1b6f9", [:mix], []},
```

Comment:
I think that the absinthe stuff should not be present in this file because they're not present in the **mix.exs** file.